packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -87,8 +109,11 @@ enum DevToolsScreenType {
   debugger,
   network,
   simple,
+  conditional,
```

Comment:
I wonder if we should drop this enum and just load screens based on a unique string id?